src/nginx_manager.py

```diff
@@ -154,8 +162,12 @@ def _create_server_config(host: str, configuration: HostConfig) -> None:
     """
     logger.info("Creating the nginx site configuration file for hosts %s", host)
     try:
-        nginx_config = nginx.Conf()
+        proxy_cache_path = NGINX_PROXY_CACHE_DIR_PATH / host
```

Comment:
Since the filename of the cache is a hash of the zone name, I don't think it's necessary to define a subfolder. All cache zones can be in the same folder.
Worse case you can define a level that will subdivide the hash into 2 characters  and will create subfolders based on them

Reply:
Removed this.